python/tank/util/qt_importer.py

```diff
@@ -280,6 +289,7 @@ def _import_pyside2_as_pyside(self):
         """
         import PySide2
         from PySide2 import QtCore, QtGui, QtWidgets
+        import shiboken2 as shiboken
```

Comment:
Is there a reason to import `shiboken2` with an alias `shiboken`? If not, I think it would be more clear to just import it as is.